docker/sawtooth-dev-javascript

```diff
@@ -14,8 +14,7 @@
 # ------------------------------------------------------------------------------
 
 # Description:
-#   Builds an image to be used when developing in JavaScript. The default CMD is to run
-#   build_javascript.
+#   Builds an image to be used when developing in JavaScript.
```

Comment:
Is there any need for this docker file to remain?

Reply:
Yes. It is used by the web socket integration test:
https://github.com/hyperledger/sawtooth-core/blob/master/integration/sawtooth_integration/docker/test_web_socket_subscription.yaml#L90